### DISCUSSION TITLE
Welcome to Invio Discussions!

### DISCUSSION THREAD

**Discussion by walkthunder** [frontend-engineering/Invio#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lvxiaoc** [comment]
good job. like the idea. havent try amazon service so cant see the result . as the export t HTML plugin can not export many things yet (like pdf,local graph) dont know how capable of this plugin does . is it possible to export to a local directory  as static html so user can quickly see the result?
